src/lib/dataviews/category/category.spec.ts

```diff
@@ -11,7 +11,8 @@ describe('DataView', () => {
         () =>
           new Category(new Layer('fake_source'), 'fake_column', {
             operation: AggregationType.AVG,
-            operationColumn: 'popEst'
+            operationColumn: 'popEst',
+            mode: DataViewModeAlias.LOCAL
```

Comment:
If we're exposing this enum, let's use just DataViewMode. ... without the 'alias'?